### ISSUE TITLE
Console messages not working?

### ISSUE THREAD

**Issue by andrei1015** [archfz/cypress-terminal-report#67]
Hello! I am trying to make console.log to show in the terminal but until now I've only managed to see cy.log. Cypress is still at version 5.5.0 and I;ve set the plugin to always display.

Tried console.log both in a test and outside, in a helper file.

**Comment by archfz** [comment]
Please provide the plugin version and the options you are using on the collector install.

**Comment by andrei1015** [comment]
Have just updated to the latest version of the plugin and rechecked.

The plugin installation:
```
module.exports = (on, config) => {
  let config = {};
  config.printLogsToConsole = 'always';
  require('cypress-terminal-report/src/installLogsPrinter')(on, config);
};
```

And the support installation:
```
let config = {};
config.collectTypes = ['cons:log', 'cons:info', 'cons:warn', 'cons:error', 'cy:log'];
require('cypress-terminal-report/src/installLogsCollector')(config);
```

And in the code I have:

```
    console.log('help');
    console.info('help');
    console.warn('help');
    console.error('help');
    cy.log('help');
```

This is in a spec file, also tried from a helper file. Only the cy.log one seems to work.

**Comment by archfz** [comment]
You shouldnt be using console logging in test code. Use cy.log there. The console logs onlt appear from your application that is tested. Cypress test execution is in primary browser frame and your site being tested is running in a child frame (iframe), only in the later we intercept console logging.

**Comment by archfz** [comment]
If for some reason you still want to use console log in test code do: `cy.window().then(w => w.console.log())`. Mainly it is not advised to use console log in test code for cypress since cypress uses a queue (async) to execute stuff in order, and plain console.log is sync.

**Comment by nicholaschiang** [comment]
@archfz Is this documented in the README somewhere? I couldn't see it and thus assumed I could use `console.log` and it would be recorded automatically.

**Comment by archfz** [comment]
No it's not documented. Let s add it in that case.

**Comment by archfz** [comment]
Released in 3.0.2
